### PR TITLE
Separate the Meet the Community into its own series of Blog Posts

### PR DESCRIPTION
**Pull request by daviddias**

### CODE REVIEW

File: content/post/86-ipfs-weekly-41.md [hunk 1]
```diff
@@ -16,35 +16,12 @@ Want this update in your inbox? [Subscribe to our weekly newsletter!](https://ti
...
-### Open Source communities are full of unsung heroes. Is there someone in IPFS community who you admire for their work?
-
-Recently, magik6k and others worked hard on refactoring the IPFS Core API and created a clean HTTP client for IPFS. I use the results of this work extensively for the experiments I’m running so I’m grateful for their work. There’s also a lot of other exciting work like hsanjuan’s IPFS driver for Docker Registry, Stebalien’s work on IPFS itself, among many others.
-
-### What are you most looking forward to at IPFS Camp?
-
-I’m most looking forward to meeting the team behind IPFS, and mind sharing with the community. It’ll also be my first time in Barcelona, so I’m excited to visit the beautiful city.
+## Meet the Community: Edgar Lee @ Software Developer at Netflix
 
+This week for [`Meet the Community`, we had Edgar Lee, a Software Developer at Netflix](INSERT URL)
```
Comment (daviddias) [inline content/post/86-ipfs-weekly-41.md]:
```suggestion
This week for [`Meet the Community`, we had Edgar Lee, a Software Developer at Netflix](https://blog.ipfs.io/meet-the-community-edgar-lee)
```

File: content/post/87-weekly-ipfs-42.md [hunk 2]
```diff
@@ -27,34 +27,9 @@ Are you keen to contribute to IPFS but don’t know where to start? Come and ask
...
+## Meet the Community: Carson Farmer, Core Developer @ Textile.io
 
+This week for [`Meet the Community`, we had Carson Farmer, a Core Developer at Textile](INSERT URL)
```
Comment (daviddias) [inline content/post/87-weekly-ipfs-42.md]:
```suggestion
This week for [`Meet the Community`, we had Carson Farmer, a Core Developer at Textile](https://blog.ipfs.io/meet-the-community-carson-farmer/)
```

File: content/post/88-ipfs-weekly-43.md [hunk 3]
```diff
@@ -16,69 +16,12 @@ Want this update in your inbox? [Subscribe to our weekly newsletter!](https://ti
...
-### Open Source communities are full of unsung heroes. Is there someone in IPFS community who you admire for their work?
-
-I really admire the work [Irakli](https://github.com/gozala/) has been doing to get p2p into Firefox with [libdweb](https://github.com/mozilla/libdweb) and his service worker explorations with [lunet](https://github.com/Gozala/lunet). He’s been really active in the IPFS community for a long time and truly understands the importance of p2p technology in the browser. He also has sound opinions about API design.
-
-### What are you most looking forward to at IPFS Camp?
-
-Science Fair. I absolutely _love_ mad science and experimenting.
-
-
+This week for [`Meet the Community`, we had Alan Shaw, a Core Developer of the IPFS Protocol @ IPFS / Protocol Labs](INSERT URL)
```
Comment (daviddias) [inline content/post/88-ipfs-weekly-43.md]:
```suggestion
This week for [`Meet the Community`, we had Alan Shaw, a Core Developer of the IPFS Protocol @ IPFS / Protocol Labs](https://blog.ipfs.io/meet-the-community-alan-shaw)
```

File: content/post/meet-the-community--Alan-Shaw-IPFS.md [hunk 4]
```diff
@@ -0,0 +1,72 @@
...
+
+I really admire the work [Irakli](https://github.com/gozala/) has been doing to get p2p into Firefox with [libdweb](https://github.com/mozilla/libdweb) and his service worker explorations with [lunet](https://github.com/Gozala/lunet). He’s been really active in the IPFS community for a long time and truly understands the importance of p2p technology in the browser. He also has sound opinions about API design.
+
+### What are you most looking forward to at IPFS Camp?
+
+Science Fair. I absolutely _love_ mad science and experimenting.
+
+--------
+
+Thank you for answering our questions, Edgar, looking forward to meet you at [IPFS Camp](https://camp.ipfs.io)
```
Comment (daviddias) [inline content/post/meet-the-community--Alan-Shaw-IPFS.md]:
```suggestion
Thank you for answering our questions, Alan, looking forward to meet you at [IPFS Camp](https://camp.ipfs.io)
```

File: content/post/meet-the-community--Carson-Farmer-Textile.md [hunk 5]
```diff
@@ -0,0 +1,40 @@
...
+
+--------
+
+Thank you for answering our questions, Edgar, looking forward to meet you at [IPFS Camp](https://camp.ipfs.io)
```
Comment (daviddias) [inline content/post/meet-the-community--Carson-Farmer-Textile.md]:
```suggestion
Thank you for answering our questions, Carson, looking forward to meet you at [IPFS Camp](https://camp.ipfs.io)
```

**Comment by daviddias** [comment]
Thank you for the review @alanshaw :)

**Comment by cwaring** [comment]
Hold your horses, I haven't added the newsletter actions yet 😂 

I'll do that in a separate PR shortly!

**Comment by daviddias** [comment]
Sounds good! This way the structure is there for Grace and Jenn to pick it up, while you add the newsletter everywhere.

**Comment by cwaring** [comment]
👍

**Pull request merged**